### PR TITLE
Allow the download of files in development

### LINKED ISSUE #70: Enable downloading raw KLEE results in development mode
Right now, trying to download KLEE's raw results in development mode is not possible.

### PR DESCRIPTION
**Pull request by helicopter88**
This fixes #70 
The URL is /jobs/dl because download was too big to fit into 80 characters

### CODE REVIEW

File: src/klee_web/worker/storage/dummy_storage.py [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
     def __init__(self):
         pass
 
     def store_file(self, file_path):
-        return 'File upload disabled in development'
+        base_name = os.path.basename(file_path)
+        final_path = os.path.join(self.DOCKER_MOUNT_DIR, base_name)
+        shutil.copy(file_path, final_path)
+        # TODO: This should be dynamic
+        return "http://192.168.33.10/jobs/dl/{0}".format(base_name)
```
Comment (andronat) [inline src/klee_web/worker/storage/dummy_storage.py]: Take a look [here](http://stackoverflow.com/questions/10119149/how-can-i-get-the-running-server-url)

File: src/klee_web/worker/storage/dummy_storage.py [hunk 2]
```diff
@@ -1,8 +1,17 @@
+import os
+import shutil
+
+
 class DummyStorage():
     BUCKET = 'klee-output'
+    DOCKER_MOUNT_DIR = '/tmp/'
```
Comment (andronat) [inline src/klee_web/worker/storage/dummy_storage.py:7]: This global is also used in `src/klee_web/frontend/views.py`. Can you find a place where we can keep global vars throughout Django and place it there?

Comment (andronat) [inline src/klee_web/worker/storage/dummy_storage.py:7]: It is also used [here](https://github.com/klee/klee-web/blob/9f16f923f9b9a3902a2f2b98f0dac4c4b47b7993/src/klee_web/worker/processor/upload.py#L29).

File: src/klee_web/worker/storage/dummy_storage.py [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
 
     def __init__(self):
         pass
 
     def store_file(self, file_path):
```
Comment (andronat) [inline src/klee_web/worker/storage/dummy_storage.py:12]: I'm a little bit confused with this method. `file_path` is a path from `self.runner.tempdir` and you then change it to `DOCKER_MOUNT_DIR`. Can't we take the file directly from `self.runner.tempdir`? Do we need to copy it?

**Comment by helicopter88** [comment]
I have no idea why, but self.runner.tempdir is not a volume shared with the host, so it is not accessible, even though it's in /tmp
There is no need to have a full URL in `store_file`, so I have removed that bit
Worker does not use django however, not sure what would happen if I were to import anything django related in it

**Comment by helicopter88** [comment]
Added some comments to clarify the usage of shutil.copy

**Pull request merged**